app/src/main/java/com/babylon/wallet/android/presentation/account/recover/complete/RecoveryScanCompleteScreen.kt

```diff
@@ -0,0 +1,96 @@
+package com.babylon.wallet.android.presentation.account.recover.complete
+
+import androidx.activity.compose.BackHandler
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.WindowInsets
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.statusBars
+import androidx.compose.material.Text
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.babylon.wallet.android.R
+import com.babylon.wallet.android.designsystem.composable.RadixPrimaryButton
+import com.babylon.wallet.android.designsystem.theme.RadixTheme
+import com.babylon.wallet.android.designsystem.theme.RadixWalletTheme
+import com.babylon.wallet.android.presentation.ui.composables.BackIconType
+import com.babylon.wallet.android.presentation.ui.composables.RadixCenteredTopAppBar
+import com.babylon.wallet.android.utils.formattedSpans
+
+@Composable
+fun RecoveryScanCompleteScreen(
+    onContinueClick: () -> Unit
+) {
+    RecoveryScanCompleteContent(onContinueClick = onContinueClick)
+}
+
+@Composable
+private fun RecoveryScanCompleteContent(
+    modifier: Modifier = Modifier,
+    onContinueClick: () -> Unit
+) {
+    BackHandler {}
+    Scaffold(
+        modifier = modifier,
```

Comment:
Maybe `.navigationBarsPadding()` is needed or else the content will be hidden behind nav bars.

Reply:
isn't scaffold handling this automatically?